src/base_obs.c

```diff
@@ -417,6 +418,13 @@ void base_obs_setup()
     &deprecated_callback,
     &deprecated_node
   );
+  
+  static sbp_msg_callbacks_node_t deprecated_node_2;
+  sbp_register_cbk(
+    SBP_MSG_OBS_DEP_B,
```

Comment:
Maybe drop `SBP_MSG_OBS_DEP_A` - and just put out a deprecated warning for `SBP_MSG_OBS_DEP_B`, not both.